examples/hybridexampleapp/package.json

```
@@ -0,0 +1,37 @@
+{
+  "name": "hybridexampleapp",
+  "version": "0.0.1",
+  "private": true,
+  "scripts": {
+    "android": "react-native run-android",
+    "ios": "react-native run-ios",
+    "lint": "eslint .",
+    "start": "react-native start",
+    "test": "jest"
+  },
+  "dependencies": {
+    "@react-native-community/cli": "^14.1.1",
+    "@react-native-community/cli-platform-android": "^14.1.1",
+    "instabug-reactnative": "^13.4.2",
```

I believe this should be imported from our local directory to better reflect current work in progress instead of the version in production like so :
```suggestion
        "instabug-reactnative": "link:../..",
```
however you would need to adjust `metro.config.js` to resolve correctly

---

yes you're right but it will use instead a package from our source code not the source code it self but this is the next task scope